mucommander-core/src/main/java/com/mucommander/ui/action/impl/RevealInDesktopAction.java

```diff
@@ -55,21 +56,25 @@ public RevealInDesktopAction(MainFrame mainFrame, Map<String, Object> properties
     @Override
     protected void toggleEnabledState() {
         AbstractFile currentFolder = mainFrame.getActivePanel().getCurrentFolder();
+        AbstractFile selectedFile = mainFrame.getActiveTable().getSelectedFile();
         setEnabled(currentFolder.getURL().getScheme().equals(LocalFile.SCHEMA)
                 && !currentFolder.isArchive()
-                && !currentFolder.hasAncestor(AbstractArchiveEntryFile.class));
+                && !currentFolder.hasAncestor(AbstractArchiveEntryFile.class)
+                || currentFolder.getURL().getScheme().equals(SearchFile.SCHEMA)
+                && selectedFile != null
+                && selectedFile.getURL().getScheme().equals(LocalFile.SCHEMA));
```

Comment:
we'd still want to disable it for archive entries that appear in the search results, right?

Reply:
I've checked that and in archives 'reveal in XYZ' doesn't appear. The condition that parent is `Search` schema and the file must be `LocalFile` should be enough. Unless I don't see sth.

Reply:
right